DistributedCollective/Sovryn-frontend
# feat: added rif token and rbtc/rif amm converter pool

Testnet contracts only, do not merge until mainnet contracts are ready too!

testing:
1. verify RIF token is in user portfolio
2. verify you can trade RIF token in swap page
3. verify you can trade RIF token in spot page
4. verify you add liquidity to RBTC/RIF amm pool (yield farming page), you will need some RIF and RBTC in your wallet (buy RIF in swap or spot page)
5. verify you can remove liquidity from RBTC/RIF amm pool after you added it in step 4.
